lib/windows/winlean.nim

```diff
@@ -33,13 +33,15 @@ type
   WINBOOL* = int32
     ## `WINBOOL` uses opposite convention as posix, !=0 meaning success.
     # xxx this should be distinct int32, distinct would make code less error prone
+  PBOOL* = ptr WINBOOL
   DWORD* = int32
   PDWORD* = ptr DWORD
   LPINT* = ptr int32
   ULONG_PTR* = uint
   PULONG_PTR* = ptr uint
   HDC* = Handle
   HGLRC* = Handle
+  BYTE* = uint8
```

Comment:
```suggestion
  BYTE* = cuchar
```
?
according to https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types